common/src/main/java/com/revenuecat/purchases/common/OfferingFactories.kt

```diff
@@ -49,15 +49,25 @@ fun JSONObject.createOffering(productsById: Map<String, StoreProduct>): Offering
     }
 }
 
+fun chooseBestOffer(products: List<StoreProduct>?): StoreProduct? {
```

Comment:
Just to make sure, we won't be choosing an offer ourselves, but creating a new model for the offers so devs can choose what offer to show right?

Reply:
Correct - this is the alternative1 implementation we used for a POC before we realized we should change the API